src/elements/content-sidebar/Sidebar.js

```diff
@@ -206,6 +253,20 @@ class Sidebar extends React.Component<Props, State> {
             'bcs-is-open': isOpen,
         });
 
+        const tablist = [];
```

Comment:
Can we pull this information from React Router, instead? The routes should be registered after the initial render. If not, does this need to be redefined in every render call? Should it have its own method?